sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ProjectReactorValidator.java

```diff
@@ -30,30 +27,27 @@
 import org.apache.commons.lang.StringUtils;
 import org.sonar.api.batch.bootstrap.ProjectDefinition;
 import org.sonar.api.batch.bootstrap.ProjectReactor;
-import org.sonar.api.config.Settings;
+import org.sonar.api.utils.MessageException;
 import org.sonar.core.component.ComponentKeys;
 import org.sonar.scanner.analysis.DefaultAnalysisMode;
 
+import com.google.common.base.Joiner;
+
 /**
  * This class aims at validating project reactor
  * @since 3.6
  */
 public class ProjectReactorValidator {
-
-  private static final String SONAR_PHASE = "sonar.phase";
-  private final Settings settings;
   private final DefaultAnalysisMode mode;
 
-  public ProjectReactorValidator(Settings settings, DefaultAnalysisMode mode) {
-    this.settings = settings;
+  public ProjectReactorValidator(DefaultAnalysisMode mode) {
     this.mode = mode;
   }
 
   public void validate(ProjectReactor reactor) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ProjectReactor"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)